translate/convert/test_po2php.py

```diff
@@ -46,6 +59,38 @@ def test_space_preservation(self):
         print(phpfile)
         assert phpfile == [phpexpected]
 
+    def test_unused_preservation(self):
+        """check that we preserve any unused statements in php files when merging"""
+        posource = '''#: $lang['name']\nmsgid "value"\nmsgstr "waarde"\n'''
```

Comment:
This is easier to read if you get rid of the `\n`. After all this is a multiline string.

Reply:
Well this is how rest of the test_po2php.py looks like, but you're right that without \n it looks better.

Reply:
We have a lot of code follows patterns that is better to avoid, so better lets start adding code following nicer patterns even if we don't refactor all legacy code.

Reply:
Once this is done, I'll probably submit some cleanup PRs as well - it cases like this it's pretty easy to start by copying other testcases and that should lead to acceptable PR...

Reply:
That's true. Improvements and cleanups are welcome. I have myself a branch refactoring all TTK converters to a common pattern that I want to start merging after your PHP changes.